src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateManager.cs

```diff
@@ -195,12 +228,15 @@ public static void UninstallAllMyCertificates(bool force)
                         // or if 'force' has asked to remove all.
                         if (force || pair.Value.AddedToStore)
                         {
-                            X509Certificate2 cert = CertificateFromThumbprint(store, pair.Value.ThumbPrint);
+                            X509Certificate2 cert = CertificateFromThumbprint(store, pair.Value.Thumbprint);
                             if (cert != null)
                             {
                                 store.Remove(cert);
-                                Console.WriteLine("Uninstalled from store '{0}', location '{1}', cert '{2}'",
-                                                    storeName, storeLocation, cert.SubjectName.Name);
+                                Trace.WriteLine(string.Format("[CertificateManager] Removed certificate from store: "));
```

Comment:
Nit: it would be better to StringBuilder() this and Trace once, using the pattern that includes a timestamp and category